pset-iterators.js

```diff
@@ -160,7 +194,12 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.find((party) => {
+  return party.party === "Republican"
+})
+
 
+console.log(firstRepublican.president)
```

Comment:
@jct117 Great Job here using the find method to return all of the presidents who belong to the Republican party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)